README.md

```diff
@@ -165,6 +166,19 @@ This rule reports an error when a function has a parameter with `null` as defaul
 
 ### Methods
 
+#### `Methods\FinalInAbstractClassRule`
+
+This rule reports an error when a concrete method in an `abstract` class is `public` or `protected`, but not final.
+
+For this rule to be enabled, the parameter `allowAbstractClasses` needs to be set to `true`:
+
+```neon
+parameters:
+	allowAbstractClasses: true
```

Comment:
Is the name correct?

Reply:
Thank you for taking a look, but yes, it's correct - it's the same parameter that is currently used by `Localheinz\PHPStan\Rules\Classes\FinalRule`, see [`rules.neon`](https://github.com/localheinz/phpstan-rules/blob/0.11.0/rules.neon#L30-L36).

On the other hand, perhaps the parameter is not even needed.